Sources/PostgresNIO/Data/PostgresData.swift

```diff
@@ -79,3 +79,17 @@ public struct PostgresData: CustomStringConvertible, CustomDebugStringConvertibl
         return self.description
     }
 }
+
+extension PostgresData: PostgresDataConvertible {
+    public static var postgresDataType: PostgresDataType {
+        return .void
```

Comment:
Unfortunately I don't think there's anything we can do here that makes sense. I think `fatalError` here is our only option. Something like:
```swift
fatalError("PostgresData cannot be statically represented as a single data type")
```

Reply:
alright fixed that :)